base/cvd/cuttlefish/host/commands/cvd/server_client.cpp

```diff
@@ -122,4 +87,92 @@ bool RequestWithStdio::IsNullIo() const {
   return &in_ == &NullIn() && &out_ == &NullOut() && &err_ == &NullOut();
 }
 
+RequestWithStdio& RequestWithStdio::OverrideRequest(RequestWithStdio other) & {
+  args_ = std::move(other.args_);
+  env_ = std::move(other.env_);
+  selector_args_ = std::move(other.selector_args_);
+  working_directory_ = std::move(other.working_directory_);
+  return *this;
+}
+
+RequestWithStdio RequestWithStdio::OverrideRequest(RequestWithStdio other) && {
+  args_ = std::move(other.args_);
+  env_ = std::move(other.env_);
+  selector_args_ = std::move(other.selector_args_);
+  working_directory_ = std::move(other.working_directory_);
+  return *this;
+}
+
+const cvd_common::Args& RequestWithStdio::Args() const { return args_; }
+
+RequestWithStdio& RequestWithStdio::AddArgument(std::string argument) & {
+  args_.emplace_back(std::move(argument));
+  return *this;
+}
+
+RequestWithStdio RequestWithStdio::AddArgument(std::string argument) && {
+  args_.emplace_back(std::move(argument));
+  return *this;
+}
+
+const cvd_common::Args& RequestWithStdio::SelectorArgs() const {
+  return selector_args_;
+}
+
+RequestWithStdio& RequestWithStdio::AddSelectorArgument(
+    std::string argument) & {
+  selector_args_.emplace_back(std::move(argument));
+  return *this;
+}
+
+RequestWithStdio RequestWithStdio::AddSelectorArgument(
+    std::string argument) && {
+  selector_args_.emplace_back(std::move(argument));
+  return *this;
+}
+
+RequestWithStdio& RequestWithStdio::ImportEnv(const cvd_common::Envs& env) & {
```

Comment:
nit: `ExtendEnv`?

Reply:
I've removed `ImportEnv`. It was used in an intermediate commit, but now `SetEnv` works as a substitute.